src/dos/dos_keyboard_layout.cpp

```diff
@@ -1322,9 +1323,51 @@ void DOS_KeyboardLayout_ShutDown(Section* /*sec*/) {
 	delete test;	
 }
 
+std::map<std::string, int> country_code_map {
```

Comment:
Table missing some supported countries like Canada and UK.

Reply:
@NicknineTheEagle I already added some missing ones like U.K to the table, although for U.S. and Canada they often have NULL keyboard keyboard layout rather than a layout name like other countries.